components/phpunit_bridge.rst

```diff
@@ -200,6 +200,26 @@ the ``vendors`` directory will make the test suite fail, while deprecations
 triggered from a library inside it will not, giving you the best of both
 worlds.
 
+Here is a summary that should help you pick the right mode:
+
++----------------+-----------------------------------------------------+
+| Mode           | Recommended situation                               |
++================+=====================================================+
+| strict         | Recommended for actively maintained projects        |
+|                | with little to no dependencies                      |
++----------------+-----------------------------------------------------+
+| <some integer> | Recommended for projects that you cannot            |
+|                | immediately fix but don't want to make worse.       |
+|                | Can be used to transition from one mode to another. |
```

Comment:
Should we have a bit more information about what integers you can use here? ie. What's the min/max? Is a lower number more strict or less strict?

Reply:
This column shows no description, even for other modes. It only shows a recommendation. That particular mode is already documented above.

Reply:
Ah fair enough. As long as it's documented somewhere :+1: